src/index.ts

```diff
@@ -9,7 +9,7 @@ export const run = () => {
   try {
     const argv = process.argv.slice(2);
     const {_: args, ...options} = minimist(argv, {
-      boolean: ['install', 'setup', 'help', 'appium', 'standalone'],
```

Comment:
We require the `standalone` flag to be boolean, otherwise, if some argument is passed after the `--standalone` flag, it will be considered as the value of the `--standalone` option.

Maybe we can just add `standalone` to the acceptable flags in ios as well for now and then figure out later how we can get rid of these flags if they are not passed in the command at all.